shared/api/endpoints/createSendChatMessageEndpoint.ts

```diff
@@ -23,5 +24,12 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<ChatMessageModel>>
       formData.append('message', message)
       return formData
     })
-    .withMapper(() => [])
```

Comment:
❓ I don't understand why the mapper here used to be empty?

Reply:
I guess because the response was not used before. There is one endpoint to send a message (this one) and another one to fetch all messages. Instead of using the return of this message, so far only the other endpoint was refetched to get the new message as well.

Reply:
Ah, I see, thanks :)